docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -44,7 +48,7 @@ apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: ovn-data-cert
-  namespace: openstack
+  namespace: {pod_ns}
```

Comment:
the use of namespace is to be explcit in our actions , so that in ospdo adoption it's clear where (rhoso or ospdo) we're running the pods and commands against.
In regular adoption it can be used and it would make either no change from the default namespace used by rhoso, or explicitly define that the user is running in the rhoso namespace, so that param only adds specificity not error if used, as in rhoso the default namespace is used anyway whether the user specifies it or not